File: fizzbuzz.py
import math
var1 = 3
var2 = 10

def main():
   i = 1
   while i<100:
      if i%(var1*var2) == 0:
         print('FizzBuzz')
         i += 1      
      if i%var1 == 0:
         print('Fizz')
         i += 1
      if i%var2 == 0:
         print("Buzz")
         i += 1
      else:
         print(i)
         i += 1

main()

File: fizzbuzz.swift
//
//  fizzbuzz.swift
//  
//
//  Created by StudentF18 on 9/10/18.
//

import Foundation
var mN = 0
while mN<100 {
    let Fizz = mN%3
    let Buzz = mN%5
    
    if Fizz == 0 {
        print("Fizz")
        mN = mN + 1
    }
    else if Buzz == 0 {
        print("Buzz")
        mN = mN + 1
    }
        //if ((Buzz == 0),(Fizz == 0)) {
        //print("FizzBuzz")
        //}
    else {
        print(mN)
        mN = mN + 1
    }
    
}
